clients/fides-js/__tests__/lib/cookie.test.ts

```diff
@@ -60,17 +57,17 @@ const mockRemoveCookie = jest.fn(
   /* eslint-disable-next-line @typescript-eslint/no-unused-vars */
   (name: string, attributes?: CookieAttributes) => undefined
 );
-jest.mock("typescript-cookie", () => ({
-  getCookie: () => mockGetCookie(),
-  setCookie: (
-    name: string,
-    value: string,
-    attributes: object,
-    encoding: object
-  ) => mockSetCookie(name, value, attributes, encoding),
-  removeCookie: (name: string, attributes?: CookieAttributes) =>
-    mockRemoveCookie(name, attributes),
-}));
+
+// @ts-ignore
+jest.spyOn(Cookies, "withConvertor").mockReturnValue(
```

Comment:
Here is where I'm struggling- trying to mock the "withConvertor" method which, in the actual lib, returns an instance of `CookiesStatic`, providing all of our `get`, `set`, etc methods.

Reply:
I've tried mocking the lib directly, too, e.g. 
```
jest.mock('js-cookie', () => ({
  __esModule: true,
  withConvertor: jest.fn(() => ({
    get: () => mockGetCookie(),
    set: (name: string, value: string, attributes: object) =>
        mockSetCookie(name, value, attributes),
    remove: (name: string, attributes?: CookieAttributes) =>
        mockRemoveCookie(name, attributes),
  }))
}));
```

Reply:
ahh turns out I just had to remove that `__esModule: true,` bit, thanks so much @gilluminate for solving this!

Reply:
so very close!